TERRAFORM CRASH on importing mongodbatlas_alert_configuration

Terraform import worked for other `mongodbatlas_alert_configuration`, but not for this one.

```
terraform import mongodbatlas_alert_configuration.replication_oplog_window 5e384f0dcf09a208528fb22d-5e384f0dcf09a208528fb22e
mongodbatlas_alert_configuration.replication_oplog_window: Importing from ID "5e384f0dcf09a208528fb22d-5e384f0dcf09a208528fb22e"...

Error: rpc error: code = Unavailable desc = transport is closing


panic: runtime error: invalid memory address or nil pointer dereference
```

[crash.log](https://github.com/terraform-providers/terraform-provider-mongodbatlas/files/4330913/crash.log)

--- a/mongodbatlas/resource_mongodbatlas_alert_configuration.go
+++ b/mongodbatlas/resource_mongodbatlas_alert_configuration.go
@@ -447,13 +447,16 @@ func expandAlertConfigurationMetricThreshold(d *schema.ResourceData) *matlas.Met
 }
 
 func flattenAlertConfigurationMetricThreshold(m *matlas.MetricThreshold) map[string]interface{} {
-	return map[string]interface{}{
-		"metric_name": m.MetricName,
-		"operator":    m.Operator,
-		"threshold":   cast.ToString(m.Threshold),
-		"units":       m.Units,
-		"mode":        m.Mode,
+	if m != nil {
+		return map[string]interface{}{
+			"metric_name": m.MetricName,
+			"operator":    m.Operator,
+			"threshold":   cast.ToString(m.Threshold),
+			"units":       m.Units,
+			"mode":        m.Mode,
+		}
 	}
+	return map[string]interface{}{}
 }
 
 func expandAlertConfigurationNotification(d *schema.ResourceData) []matlas.Notification {

--- a/mongodbatlas/resource_mongodbatlas_alert_configuration_test.go
+++ b/mongodbatlas/resource_mongodbatlas_alert_configuration_test.go
@@ -146,6 +146,7 @@ func TestAccResourceMongoDBAtlasAlertConfiguration_whitMetricUpdated(t *testing.
 		},
 	})
 }
+
 func TestAccResourceMongoDBAtlasAlertConfiguration_importBasic(t *testing.T) {
 	projectID := os.Getenv("MONGODB_ATLAS_PROJECT_ID")
 
@@ -158,6 +159,30 @@ func TestAccResourceMongoDBAtlasAlertConfiguration_importBasic(t *testing.T) {
 		Steps: []resource.TestStep{
 			{
 				Config: testAccMongoDBAtlasAlertConfigurationConfig(projectID, true),
+			},
+			{
+				ResourceName:            resourceName,
+				ImportStateIdFunc:       testAccCheckMongoDBAtlasAlertConfigurationImportStateIDFunc(resourceName),
+				ImportState:             true,
+				ImportStateVerify:       true,
+				ImportStateVerifyIgnore: []string{"project_id"},
+			},
+		},
+	})
+}
+
+func TestAccResourceMongoDBAtlasAlertConfiguration_importConfigNotifications(t *testing.T) {
+	projectID := os.Getenv("MONGODB_ATLAS_PROJECT_ID")
+
+	resourceName := "mongodbatlas_alert_configuration.test"
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testAccCheckMongoDBAtlasAlertConfigurationDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccMongoDBAtlasAlertConfigurationConfigNotifications(projectID, true, true, false),
 			},
 			{
 				ResourceName:            resourceName,
@@ -228,7 +253,7 @@ func testAccMongoDBAtlasAlertConfigurationConfig(projectID string, enabled bool)
 			project_id = "%s"
 			event_type = "OUTSIDE_METRIC_THRESHOLD"
 			enabled    = "%t"
-		
+
 			notification {
 				type_name     = "GROUP"
 				interval_min  = 5
@@ -236,13 +261,13 @@ func testAccMongoDBAtlasAlertConfigurationConfig(projectID string, enabled bool)
 				sms_enabled   = false
 				email_enabled = true
 			}
-		
+
 			matcher {
 				field_name = "HOSTNAME_AND_PORT"
 				operator   = "EQUALS"
 				value      = "SECONDARY"
 			}
-		
+
 			metric_threshold = {
 				metric_name = "ASSERT_REGULAR"
 				operator    = "LESS_THAN"
@@ -260,7 +285,7 @@ func testAccMongoDBAtlasAlertConfigurationConfigNotifications(projectID string, 
 			project_id = "%[1]s"
 			event_type = "NO_PRIMARY"
 			enabled    = "%[2]t"
-		
+
 			notification {
 				type_name     = "GROUP"
 				interval_min  = 5
@@ -286,7 +311,7 @@ func testAccMongoDBAtlasAlertConfigurationConfigWithMatchers(projectID string, e
 			project_id = "%s"
 			event_type = "NO_PRIMARY"
 			enabled    = "%t"
-		
+
 			notification {
 				type_name     = "GROUP"
 				interval_min  = 5
@@ -294,7 +319,7 @@ func testAccMongoDBAtlasAlertConfigurationConfigWithMatchers(projectID string, e
 				sms_enabled   = %t
 				email_enabled = %t
 			}
-		
+
 			matcher {
 				field_name = "%s"
 				operator   = "%s"
@@ -305,7 +330,7 @@ func testAccMongoDBAtlasAlertConfigurationConfigWithMatchers(projectID string, e
 				operator   = "%s"
 				value      = "%s"
 			}
-		}	
+		}
 	`, projectID, enabled, smsEnabled, emailEnabled,
 		m1.FieldName, m1.Operator, m1.Value,
 		m2.FieldName, m2.Operator, m2.Value)
@@ -317,7 +342,7 @@ func testAccMongoDBAtlasAlertConfigurationConfigWithMetrictUpdated(projectID str
 			project_id = "%s"
 			event_type = "OUTSIDE_METRIC_THRESHOLD"
 			enabled    = "%t"
-		
+
 			notification {
 				type_name     = "GROUP"
 				interval_min  = 5
@@ -325,13 +350,13 @@ func testAccMongoDBAtlasAlertConfigurationConfigWithMetrictUpdated(projectID str
 				sms_enabled   = false
 				email_enabled = true
 			}
-		
+
 			matcher {
 				field_name = "HOSTNAME_AND_PORT"
 				operator   = "EQUALS"
 				value      = "SECONDARY"
 			}
-		
+
 			metric_threshold = {
 				metric_name = "ASSERT_REGULAR"
 				operator    = "LESS_THAN"

